package.json

```diff
@@ -27,10 +27,11 @@
     "clsx": "^1.2.1",
     "cspell": "^6.31.1",
     "klaw": "^4.1.0",
+    "mdx-mermaid": "^v1.3.2",
     "medium-zoom": "^1.0.8",
-    "mermaid": "^10.0.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "mermaid": "^10.0.2",
+    "react": "^18.2.0",
```

Comment:
I don't think there is ofifcial support for react 18 yet https://github.com/facebook/docusaurus/issues/7264

are we sure about this update?

Reply:
No, i don't remember the reason of this update, but I don't notice any problem. 
However to make sure that it don't cause a mess, i prefer to reset this changes